setup.cfg

```diff
@@ -45,9 +45,7 @@ install_requires =
     joblib 
     matplotlib 
     numba
-    numpy
     protozfits >=2.5,<3
-    scipy >=1.8, <1.12
```

Comment:
I think the whole mess is related to limitations on the scipy version, because in environment file there is no limit on the version, and then downgrading scipy from 1.13.1 to 1.11.4 probably required downgrading also numpy.
Maybe we simply skip the limits? Or if they are needed they can be put in the environment file as well